js/components/LoginScreen.js

```diff
@@ -166,6 +169,14 @@ class LoginScreen extends React.Component {
         />
         <View style={{ height: 10 }} />
         <SpidSubscribeComponent />
+        <View style={styles.paddingButton} />
+        <Button
+          transparent
+          block
+          onPress={() => this.props.navigation.navigate(ROUTES.NO_SPID)}
+        >
+          <Text style={{ color: 'white', fontSize: 16 }}>Non hai Spid?</Text>
```

Comment:
text must go into translations - @fpersico can we make this as a rule? all text must be translatable